Tools/LambdaTestTool-v2/tests/Amazon.Lambda.TestTool.UnitTests/SnapshotHelper/ApiGatewayTestSnapshot.cs

```diff
@@ -0,0 +1,70 @@
+using System.Net;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Amazon.Lambda.TestTool.UnitTests.SnapshotHelper;
+
+public class HttpResponseMessageConverter : JsonConverter<HttpResponseMessage>
```

Comment:
add docs explaining what this is for and what is is doing

Reply:
added docs in latest commit